app-editors/gnome-latex/gnome-latex-3.42.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2
+
+DESCRIPTION="Integrated LaTeX environment for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/GNOME-LaTeX https://gitlab.gnome.org/swilmet/gnome-latex"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="+introspection +latexmk rubber"
+
+DEPEND="
+	>=app-text/gspell-1.8:0=
+	>=dev-libs/glib-2.56:2
```

Comment:
I don't know what this diff means from `configure.ac`, but I suspect we should just depend on glib-2.70 since it's been in tree for plenty of time now:

````diff
+GLIB_TARGET_VERSION_FOR_VALA="2.70"
+AC_SUBST([GLIB_TARGET_VERSION_FOR_VALA])
````